From 186b9ad1ee3e01acd8e3ff149fb229bf10aa743e Mon Sep 17 00:00:00 2001
From: Vitaly Margolin <vmargolin@habana.ai>
Date: Tue, 23 Jan 2024 16:25:55 +0200
Subject: [PATCH] [SW-166734] gaudi2: remove arpm ppboot compilation

PPBOOT on arm is not in use, so it is not built

Signed-off-by: Vitaly Margolin <vmargolin@habana.ai>
Change-Id: Iaff4286b24cb66e70bf803f1fbda00a9bcdeba69
---

diff --git a/configs/gaudi2_arm_defconfig b/configs/gaudi2_arm_defconfig
index a96963a..fe853ac 100644
--- a/configs/gaudi2_arm_defconfig
+++ b/configs/gaudi2_arm_defconfig
@@ -60,7 +60,7 @@
 BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SUPPORT=y
 BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SIGNATURE_SUPPORT=y
 BR2_PACKAGE_ARMCP=y
-BR2_PACKAGE_PREBOOT=y
+BR2_PACKAGE_PREBOOT=n
 BR2_PACKAGE_BTL=y
 BR2_PACKAGE_EEPROM_UTIL=y
 BR2_POST_IMAGE_UBOOT_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/uboot_extended.its"
